DISCOVER/conf.py

```diff
@@ -3,6 +3,40 @@
 # If you wish to continue using _config.yml, make edits to that file and
 # re-generate this one.
 ###############################################################################
+
+version = '2.0'
```

Comment:
```suggestion
version = 'dev'
```

so we don't end up forgotting and merge this into `main`